ui/tests/integration/gauge-chart-test.js

```diff
@@ -0,0 +1,53 @@
+import { find, render } from '@ember/test-helpers';
+import { module, test } from 'qunit';
+import { setupRenderingTest } from 'ember-qunit';
+import hbs from 'htmlbars-inline-precompile';
+import { create } from 'ember-cli-page-object';
+import gaugeChart from 'nomad-ui/tests/pages/components/gauge-chart';
+
+const GaugeChart = create(gaugeChart());
+
+module('Integration | Component | gauge chart', function(hooks) {
+  setupRenderingTest(hooks);
+
+  const commonProperties = () => ({
+    value: 5,
+    total: 10,
+    label: 'Gauge',
+  });
+
+  test('presents as an svg, a formatted percentage, and an svg', async function(assert) {
```

Comment:
Is this meant to be “and a label” maybe?

Reply:
fixed.